### PR TITLE
Add the support of using kernelflinger to boot in NUC.

### PR DESCRIPTION
**Pull request by tanminger**
Jira: None.
Test: Test it in SKL NUC, can boot to UI.

Change-Id: Id41e8b25bc403bda1f3f505be28f0c8f8721111b
Signed-off-by: Ming Tan <ming.tan@intel.com>

### CODE REVIEW

File: initrd/scripts/2-install [hunk 1]
```diff
@@ -70,6 +79,10 @@ select_device()
 		read ans
 		if [ $ans -gt 0 -a $ans -le $i ]; then
 			device=`echo $list | cut -d" " -f$ans`
+			if [ $install_kf -eq 1 -a "$device" != "sda" ]; then
```
[the remark this answers is not in the document]
Comment (tanminger) [reply initrd/scripts/2-install]: This installer is used for SSD or SATA, not for eMMC.
Currently use kernelflinger in NUC only support /dev/sda.

[the remark this answers is not in the document]
Comment (harishkrupo) [reply initrd/scripts/2-install]: Why not support it if we can? :)

[the remark this answers is not in the document]
Comment (tanminger) [reply initrd/scripts/2-install]: There is not clear requirement for support it.
However, if someone has interesting for it, he can modify it to add this support :)

[the remark this answers is not in the document]
Comment (harishkrupo) [reply initrd/scripts/2-install]: ok, then seems fine. :+1:

**Comment by tanminger** [comment]
Resolve the conflict file. https://github.com/android-ia/bootable-live-installer/pull/31

**Pull request closed without merging**